set.mm

```diff
@@ -838383,6 +838391,23 @@ additional condition (deduction form).  See ~ ralbidc for a more
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Indexed union and intersection
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    $d A y $.  $d B y $.  $d x y $.
+    $( An indexed intersection is empty if there is an empty set.  (Contributed
```

Comment:
"if there is an empty set. " sounds strage => maybe better "if one of the intersected classes is empty"